shell/components/CodeMirror.vue

```diff
@@ -123,12 +123,16 @@ export default {
   },
 
   async mounted() {
-    document.addEventListener('keyup', this.handleKeyPress);
```

Comment:
Thanks for updating this 🙌. Removing listeners on the document will help prevent conflicts.